src/rust/dbv2/migrations/2023-09-04-113144_postgrest/up.sql

```diff
@@ -0,0 +1,29 @@
+-- Your SQL goes here
+CREATE ROLE web_anon nologin;
+
+
+CREATE SCHEMA api;
+
+
+CREATE VIEW
+  api.market_registration_events AS
+SELECT
+  *
+FROM
+  market_registration_events;
+
+
+GRANT usage ON SCHEMA api TO web_anon;
+
+
+GRANT
+SELECT
+  ON api.market_registration_events TO web_anon;
+
+
+CREATE EXTENSION pgjwt CASCADE;
+
+
+CREATE FUNCTION api.jwt (json) RETURNS TEXT AS $$
```

Comment:
I don't really know what 'econia_is_dope' is doing, and whether or not that's a safe/good way to do the thing.

Reply:
The thing is the DB is open. We don't want to issue API keys right now. So I just made a JWT generator that literally gives you full authorizations for querying. A JWT is necessary to use with `postgres-websockets`.